web/src/main/java/org/cbioportal/web/util/InvolvedCancerStudyExtractorInterceptor.java

```diff
@@ -103,6 +104,7 @@
     public static final String TREATMENT_FETCH_PATH = "/treatments/fetch";
     public static final String STRUCTURAL_VARIANT_FETCH_PATH = "/structuralvariant/fetch";
     public static final String GENERIC_ASSAY_DATA_MULTIPLE_STUDY_FETCH_PATH = "/generic_assay_data/fetch";
+    public static final String GENERIC_ASSAY_META_FETCH_PATH = "/generic_assay_meta/fetch";
```

Comment:
@dippindots just noticed these `gneric_assay` paths have underscores in the name while other paths do not. Prolly good to be consistent?

Reply:
we can fix it later but just FYI